patches/kernelci-pipeline/staging.kernelci.org/0001-STAGING-config-staging.kernelci.org.conf-add-file-fo.patch

```diff
@@ -39,7 +39,7 @@ index 0000000..d4752be
 +
 +[send_kcidb]
 +kcidb_topic_name: playground_kcidb_new
-+kcidb_project_id: kernelci-api
++kcidb_project_id: kernelci-production
```

Comment:
Ah so I guess these are dictated by KCIDB then.

@spbnick What are the project IDs that we can use?  I guess we can just change the name of the "origin" to distinguish what the new API / pipeline are sending from staging.kernelci.org?